src/utils/index.js

```diff
@@ -97,6 +133,28 @@ export function getWeeksInMonth(
   return rowCount;
 }
 
+/**
+ * Get the number of weeks in a given month to be able to calculate the height of that month
+ * @param {Number} year - the year number
+ * @param {Number} month - the index of the month
+ * @param {Number} weekStartsOn - the index of the first day of the week (from 0 to 6)
+ * @return {Number} - Returns the number of weeks for the given month
+ */
+export function getWeeksInMonthNoPadding(
```

Comment:
it almost feels like this is `getWeeksInMonthWithPaddingAlways` because the other one sometimes pads and sometimes doesn't, and this one always adds 1 at the end.

i'm not sure what a better name is, but it does feel a little weird to me that in the case where you're using padding you call the `NoPadding` version.

Reply:
Good call. I will update the game.